scripts/ci/lint-python.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1091
+#
+# Reformats Python code belonging to the repository.
+# The primary Linter with support for reformatting
+# files is Black. Any reformatted code causes
+# non-zero exit code (CI failure).
+# Additionally, there can be secondary Linters
+# used to raise warnings, e.g., Flake8
+
+REPOROOT=$(realpath "$(dirname "$0")/../..")
+PATTERN="./*.py"
+LINTER=$(command -v black)
+
+if [ "${LINTER}" ]; then
```

Comment:
This looks very similar to the other one. There's strong incentive for refactoring them into functions and just using them from here.

Reply:
I would defer this to the "Shellcheck-task" if possible. So far, little is duplicated and amount of code for clang-format is only 2/3rds.

Reply:
Ahh, you meant the early exit.